morecantile/models.py

```diff
@@ -669,7 +690,7 @@ def tiles(
 
         for w, s, e, n in bboxes:
             for z in zooms:
-                ul_tile = self.tile(w, n, z)
+                ul_tile = self.tile(w + LL_EPSILON, n - LL_EPSILON, z)
```

Comment:
⚠️ this resolves the `round trip` issue which was returning 6 tiles instead of 4. this was I believe a precision issue

Reply:
```python
# Before 
list(tms.tiles(*tms.bounds(3413, 6202, 14), zooms=[15]))
>>> [Tile(x=6825, y=12404, z=15),
 Tile(x=6825, y=12405, z=15),
 Tile(x=6826, y=12404, z=15),
 Tile(x=6826, y=12405, z=15),
 Tile(x=6827, y=12404, z=15),
 Tile(x=6827, y=12405, z=15)]

# Now
list(tms.tiles(*tms.bounds(3413, 6202, 14), zooms=[15]))

>>> [Tile(x=6826, y=12404, z=15),
 Tile(x=6826, y=12405, z=15),
 Tile(x=6827, y=12404, z=15),
 Tile(x=6827, y=12405, z=15)]
```